schemas/equilibrium/dd_equilibrium.xsd

```diff
@@ -2820,7 +2820,7 @@
 			<xs:appinfo>
 				<lifecycle_status>active</lifecycle_status>
 				<lifecycle_version>3.1.0</lifecycle_version>
-				<lifecycle_last_change>3.42.0</lifecycle_last_change>
+				<lifecycle_last_change>4.1.0</lifecycle_last_change>
```

Comment:
I'm not sure this change in this PR should affect the lifecycle last change version so I assume this is to fix a previous omission?